tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3328,8 +3360,15 @@ def runTest(self):
             # recv port no pfc
             assert (recv_counters[pg] == recv_counters_base[pg])
             # recv port no ingress drop
+            # For dnx few extra ipv6 NS/RA pkt received, adding to coutner value
+            # & may give inconsistent test results
+            # Adding COUNTER_MARGIN to provide room to 2 pkt incase, extra traffic received
             for cntr in ingress_counters:
                 if platform_asic and platform_asic == "broadcom-dnx" and cntr == 1:
+                    print("recv_counters_base: %d, recv_counters: %d" % (recv_counters_base[cntr],
+                                                                         recv_counters[cntr]), file=sys.stderr)
+                    assert (recv_counters[cntr] <= recv_counters_base[cntr] + COUNTER_MARGIN)
+                else:
```

Comment:
@saksarav-nokia @ansrajpu-git This seems to be added for non chassis. Is this intentional ?

Reply:
Looks like this is fixing mistake from previous commit. Looks good.